internal/services/compute/orchestrated_virtual_machine_scale_set_extensions_resource_test.go

```diff
@@ -101,8 +153,10 @@ resource "azurerm_orchestrated_virtual_machine_scale_set" "test" {
   location            = azurerm_resource_group.test.location
   resource_group_name = azurerm_resource_group.test.name
 
-  sku_name  = "Standard_D1_v2"
-  instances = 1
+  sku_name = "Standard_D1_v2"
+
+  # Orchestrated VMSS allocation will timeout at service side due to extension, set instances to 0 to avoid the timeout
+  instances = 0
```

Comment:
This sounds like a bug as it timesout not errors 0- can we open an issue on the rest specs and link it here. Then add a note to the docs and maybe add a check in code to throw an error if the user doesn't correctly do this?

Reply:
This appears to be a known issue and has been [documented](https://docs.microsoft.com/en-us/troubleshoot/azure/virtual-machine-scale-sets/vm-extension-provisioning-errors) as being the extensions issue.